### PR TITLE
fix cve

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: local-static-provisioner.yaml [hunk 1]
```diff
@@ -29,14 +29,14 @@ pipeline:
 
   - uses: go/bump
     with:
-      deps: google.golang.org/protobuf@v1.33.0 golang.org/x/net@v0.23.0
+      deps: google.golang.org/protobuf@v1.33.0 golang.org/x/net@v0.23.0 k8s.io/kubernetes@v1.27.13
```
Comment (hectorj2f) [inline local-static-provisioner.yaml:32]: Try adding `k8s.io/apiserver@v0.27.13`

Comment (developer-guy) [reply local-static-provisioner.yaml]: it worked, but how did you know that 😮

Comment (developer-guy) [reply local-static-provisioner.yaml]: tell me your secret sauce 🙈

Comment (hectorj2f) [reply local-static-provisioner.yaml]: Fixing way too many CVEs 😆 !

**Review by hectorj2f — APPROVED** [review]

**Pull request merged**